tests/kernel/fpu_sharing/generic/prj.conf

```diff
@@ -3,3 +3,4 @@ CONFIG_FPU=y
 CONFIG_FPU_SHARING=y
 CONFIG_STDOUT_CONSOLE=y
 CONFIG_CBPRINTF_FP_SUPPORT=y
+CONFIG_CBPRINTF_NANO=y
```

Comment:
This is wrong: the test requires floating point support to display the value of PI.  The nano formatter doesn't support floating point, so with this change it prints:

    Pi calculation OK after 50 (high) + 1 (low) tests (computed %.6lf)

Reply:
Thanks @pabigot , so the solution shall be increase the main stack size.